src/app/containers/PageHandlers/withContexts.jsx

```diff
@@ -69,6 +73,8 @@ const WithContexts = Component => {
     service: string.isRequired,
     variant: variantPropType,
     timeOnServer: number,
+    // eslint-disable-next-line react/forbid-prop-types
+    remoteConfig: object,
```

Comment:
Is there not a specific format for the `remoteConfig` we can define with `shape` here, or create our own propType for its format?

Reply:
I think

```
objectOf(
  shape({
    enabled: bool.isRequired,
    value: string,
  }),
)
```

would work?

Reply:
This seems to work, but I'm not sure it's correct, the introduction of the `_environment` key from another change has complicated it a little
```
    toggles: oneOfType([
      shape({
        _environment: string.isRequired,
      }),
      shape({
        enabled: bool.isRequired,
        value: string,
      }),
    ]).isRequired,
```